deploy/services-demo/conf/nginz/nginx-docker.conf

```diff
@@ -355,6 +355,22 @@ http {
       proxy_pass http://spar;
     }
 
+    # Stern Endpoints
+
+    # We add a `/stern` suffix to the URL to resolve clashes with non-Stern endpoints.
+    rewrite ^/backoffice/api-docs/stern  /stern/api-docs?base_url=http://127.0.0.1:8080/stern/ break;
+
+    location /stern/api-docs {
+      include common_response_no_zauth.conf;
+      proxy_pass http://stern;
+    }
+
+    # The trailing slash matters, as it makes sure the `/stern` prefix is removed.
```

Comment:
you changed this in `deploy/services-demo/conf/nginz/nginx.conf`, are you sure it should stay like this here?